src/metadata.ts

```diff
@@ -18,34 +18,12 @@
 import metadata from '../public/data/VSI_metadata.json';
  
 export function fetchCountryMetaData(countryNameString: string) {
-    //const countryData = metadata;
     return metadata.filter(c => c.countryName === countryNameString);
 }
 
 export function fetchCountryNames() {
     // TODO: we can add some logic here and in the metadata.json to filter for a valid country tag.
     var countryList = [];
     metadata.forEach((country) => countryList.push(country.countryName));
-    // TODO: remove filter CA or IE when it is ready for launch
-    return countryList.filter(c => c != "Canada");
+    return countryList;
```

Comment:
This can be simplified to one expression and is idiomatic to the way Typescript is written:
```
return metadata.map(({countryName}) => countryName);
```